### ISSUE TITLE
3.0 - Orders - Renamed Product Name Not Accessible

### ISSUE THREAD

**Issue by spencerfinnell** [easydigitaldownloads/easy-digital-downloads#7572]
### Expected behavior

When a Download has been renamed the Order details should make the new name of the Download available to screen readers.

### Actual behavior

The information about the new name is under an EDD tooltip, which is not announced by screen readers.

### Steps to reproduce the behavior
1) Purchase a download.
2) Rename the download.
3) View the Order record.
4) Tab through the Order Items table and listen for announcements.

### Information (if a specific version is affected):
EDD Version (or branch): `release/3.0`

**Comment by spencerfinnell** [comment]
PR #7573 ensures a screen reader will announce the name change.

It's still not 100% keyboard accessible because EDD tooltips are not focusable and therefore cannot be toggled by the keyboard alone. This will need to be dealt with separately project-wide.
